src/animations.ts

```diff
@@ -478,13 +478,31 @@ function setRadialProgressBar(animation: AnimationInfo, animationTime: number) {
 }
 
 
+function startSampleQuestionAnimation(animationName: AnimationName) {
+    if (CURRENT_ANIMATION_INFO.name !== animationName) {
```

Comment:
The all caps for `CURRENT_ANIMATION_INFO` can be misleading if this is not actually a constant